FormViewController

fwpushan opened the pull request:
Create and update test for generic FormCollectionViewController.

fwpushan committed:
Changes
-------------
1. Start Development of FormCollectionViewController
2. Create Infrastructure for Dynamic FormCell Types
--------

fwpushan committed:
FormViewController
--------------------------
1. Create generic FormCollectionViewController
2. Create cell type
3. Create Unit test

fwpushan committed:
Merge branch 'dev' into 315-plantObservation

fwpushan committed:
Changes
------------
1. Add ignoredProperty for FormModel
2. Update tests
3. Create Code table object

fwpushan committed:
Changes
--------------
1. Separate CodeTable to a new file
2. Add functional extension of code tables
3. Add code table displayable strings property

AmirShayegh commented on InvasivesBC/Form Framework/Core/FormViewController/FormCollectionViewController.swift:
```diff
@@ -0,0 +1,237 @@
+    }
+
+    /*
+    // MARK: - Navigation
```
this can be removed I guess

fwpushan committed:
Final Commit
----------
1. Create unit test for DropDown and Utility
2. Update CodeTable and Field model

fwpushan committed:
Changes
------
1. Update Field and CodeTable models
2. Update utility tests
-----
